webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js

```diff
@@ -132,4 +134,24 @@ describe('Test utils', () => {
         });
     });
   });
+  
+  it('Check that all test specs belong to a test suites', () => {
+    const testFolders = [];
+    for (const [, value] of Object.entries(constants.SUITES)) {
```

Comment:
Is it necessary to pull out suites into `constants`? Is importing directly from `wdio.conf.js` not possible? 
If yes, I think it would be better to store then separately from `constants`, since it's only used by this config file. Generally we could consider breaking down `wdio.conf.js` to be more modular. It's a big file at the moment.